packages/docs/stories/components/button.stories.ts

```diff
@@ -142,6 +142,56 @@ export const SettingACustomWidth: Story = {
   <syn-button size="large" style="width: 100%;">Large</syn-button>`,
 };
 
+export const IconOnly: Story = {
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('button', 'icon-only'),
+      },
+    },
+  },
+  render: () => html`
+    <syn-button size="small" variant="filled">
+      <syn-icon name="settings"></syn-icon>
```

Comment:
Please add a label on every `syn-icon` to make this accessible.

Reply:
Thanks! Added the labels